src/components/organisms/sidebar/userMenu.tsx

```diff
@@ -31,6 +36,23 @@ export const UserMenu = ({ openFeedbackForm }: { openFeedbackForm: () => void })
 		close();
 	};
 
+	const getStatusIndicatorClasses = (status?: MemberStatus) =>
+		cn("absolute right-1 top-1/2 size-2.5 -translate-y-1/2 rounded-full hidden", {
+			"bg-error-200 block": status === MemberStatus.invited,
+		});
```

Comment:
What if we change like this:
```suggestion
		cn("absolute right-1 top-1/2 size-2.5 -translate-y-1/2 rounded-full invisible", {
			"bg-error-200 visible": status === MemberStatus.invited,
		});
```

Would it be a better solution?